package.json

```diff
@@ -50,7 +52,10 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "check-lint": "prettier --list-different '**/*.js' && eslint '**/*.js'",
-    "lint": "prettier --write '**/*.js' && eslint --fix '**/*.js'"
+    "lint": "prettier --write '**/*.js' && eslint --fix '**/*.js'",
+    "lint": "npm run lint:js && npm run lint:css",
+    "lint:js": "eslint '**/*.{js}'",
```

Comment:
```suggestion
    "lint": "npm run lint:js && npm run lint:css",
    "lint:js": "prettier --write '**/*.js' && eslint --fix '**/*.js'",
```

Reply:
Note that the ESLint is configured to _fix_ the problems, while Stylelint is configured to _find_ the problems.

I don't think it matters that these don't match, but we could optionally add the `--fix` option to the Stylelint command as well.